acceso/templates/acceso/caso_index.html

```diff
@@ -11,41 +11,6 @@
 {% endblock %}
 
 {% block content %}
-    <div id="jsGrid"></div>
+<div id="jsGrid"></div>
+<script src="{% static 'acceso/js/caso_index.js' %}"></script>
 {% endblock %}
-
-{% block scripts %}
-    <script>
-        $(function() {
```

Comment:
Moved all of this into `acceso/static/acceso/js/caso_index.js`. HTML gets really hard to read when it's cluttered with big inline scripts like this. In general, if it's more than 5 lines of JS it should go in a separate file.